src/plugin.js

```diff
@@ -173,33 +175,34 @@ const onPlayerReady = (player, options) => {
         : <div class="vjs-errors-message">${player.localize(error.message)}</div>
         </div>`;
     }
-
     display = player.errorDisplay;
```

Comment:
Did we always do that? Maybe it's better to not re-use the videojs error display directly but rather create a new error display component?